docs/internationalization.md

```diff
@@ -0,0 +1,113 @@
+# Internationalization
+
+We want the official Bluesky app to be supported in as many languages as possible. If you want to help us translate the app, please reach out to us at info@blueskyweb.xyz
```

Comment:
Is that email right? Either way, we need to replace this with the actual instructions once we have them

Reply:
That is fair. I will just ask people to create a github PR or issue for now instead of the email.